### PR TITLE
[hi] Localize content/hi/docs/reference/glossary/job.md

### PR DESCRIPTION
**Pull request by amolmote**
Signed-off-by: amolmote <amolmote201@gmail.com>
Fixes #35600 [hi] Localize content/hi/docs/reference/glossary/job.md

### CODE REVIEW

File: content/hi/docs/reference/glossary/job.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+tags:
+- fundamental
+- core-object
+- workload
+---
+ एक परिमित या बैच कार्य जो पूरा होने तक चलता है।
+
+<!--more--> 
+
+एक या अधिक {{<glossary_tooltip term_id="pod" >}} ऑब्जेक्ट बनाता है और सुनिश्चित करता है कि उनमें से एक निर्दिष्ट संख्या सफलतापूर्वक समाप्त हो जाए। जैसे ही पॉड्स सफलतापूर्वक पूरा होता है, जॉब सफल पूर्णता को निगाह रखता है।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/job.md:19]: The first part of this sentence sounds incomplete in Hindi. Please could we go beyond the machine translation and actually translate this sentence?

**Review by divya-mohan0209 — COMMENTED** [review]
Left feedback.

File: content/hi/docs/reference/glossary/job.md (hunk 1 above)
Comment (amolmote) [reply content/hi/docs/reference/glossary/job.md:19]: sure @divya-mohan0209 ,I will try to improve it again

Comment (amolmote) [reply content/hi/docs/reference/glossary/job.md]: @divya-mohan0209 , I have made some changes. please check and give your suggestion

File: content/hi/docs/reference/glossary/job.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+tags:
+- fundamental
+- core-object
+- workload
+---
+ एक परिमित या बैच कार्य जो पूरा होने तक चलता है।
+
+<!--more--> 
+
+{{<glossary_tooltip term_id="pod" >}} यह एक या अधिक ऑब्जेक्ट बनाता है और सुनिश्चित करता है कि उनमें से एक निर्दिष्ट संख्या सफलतापूर्वक समाप्त हो जाए। जैसे ही पॉड्स सफलतापूर्वक पूर्ण होता है, जॉब सफल समापन होने का पता करते है।
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/job.md:19]: Is the sentence now better to read/understand? Thoughts, @divya-mohan0209 and @amolmote? I am also open to suggestions by other reviewers as well. Thanks
```suggestion
जॉब एक या अधिक {{<glossary_tooltip text="पॉड" term_id="pod" >}} ऑब्जेक्ट बनाता है और सुनिश्चित करता है कि उनमें से एक निर्दिष्ट संख्या सफलतापूर्वक समाप्त हो जाए। जैसी ही पॉड्स सफलतापूर्वक पूर्ण होते हैं, जॉब उस सफल समापन को ट्रैक करता है।
```

**Comment by amolmote** [comment]
#38132 opened  new PR for this issue due to some issues.

**Pull request closed without merging**